saleor/attribute/tests/fixtures/attribute.py

```diff
@@ -487,8 +487,12 @@ def numeric_attribute(db):
         filterable_in_dashboard=True,
         available_in_grid=True,
     )
-    AttributeValue.objects.create(attribute=attribute, name="9.5", slug="10_5")
-    AttributeValue.objects.create(attribute=attribute, name="15.2", slug="15_2")
+    AttributeValue.objects.create(
```

Comment:
I'm not sure about storing duplicated values on one model. In this implementation, we store a numeric value in `name` and `numeric` fields. It increases the chances of having an AttributeValue with a different `name` and `numeric` value. Maybe we should add some database constraints?
@maarcingebala what do you think?

Reply:
I think this is expected since "name" is meant for a "printable" version of the attribute's value. It shouldn't be included in any logic, etc., just representation (IMO).